lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1412,9 +1413,16 @@ HdDirtyBits HdVP2BasisCurves::_PropagateDirtyBits(HdDirtyBits bits) const
     // Propagate dirty bits to all draw items.
     for (const std::pair<TfToken, HdReprSharedPtr>& pair : _reprs) {
         const HdReprSharedPtr& repr = pair.second;
+#if HD_API_VERSION < 35
         const HdRepr::DrawItems& items = repr->GetDrawItems();
```

Comment:
Would simple const auto& work in this case?

Reply:
Yeah, I think it probably would.

I'm not the original author of these changes so I'm not certain about their intent, but my guess is that they chose not to use `auto` in order to make the change in API more obvious to the reader. I can pull the `GetDrawItems()` call out of the conditional and use `const auto&` instead if you prefer that though.

Reply:
Addressed by commit https://github.com/Autodesk/maya-usd/pull/534/commits/fd9aef010ea3702625668fa869e67dec2d2ab962.

Reply:
I was actually hoping it will make a bigger change...but didn't notice pointer vs reference difference. I will give up on it & thx for trying.